### ISSUE TITLE
Rendering issue Italy

### ISSUE THREAD

**Issue by MrReSc** [der-stefan/OpenTopoMap#152]
Hi
First of all, thank you for your maps. I'm using it very often.
Right now, I'm using the italian garmin map. As you can see on the pictures on the northern part of sadegna the blue colour of the see is missing.

https://imagebin.ca/v/3yZg5P4J9Riu

**Comment by der-stefan** [comment]
Yes, that is the good old issue with the sea polygons: Either missing, or flooding the land. We haven't done any development on the Garmin maps for nearly four years - so please be patient. Nobody of us can spend the time that would be necessary to do a complete update of the Garmin build process.

**Comment by MrReSc** [comment]
No problem. I just wanted to let you know.

**Comment by brivars** [comment]
Is there any update on this? What is causing it? Can it have something to do with some fault in the data from OSM or is it purely a rendering issue for the OTM Garmin version?
I'm having this on the west coast of Latvia too.
![LatviaWCflood1](https://user-images.githubusercontent.com/26169578/96759792-2ee10400-13e1-11eb-9d3c-ac99a3a29e66.png)
![LatviaWCflood2](https://user-images.githubusercontent.com/26169578/96759796-2f799a80-13e1-11eb-8f4a-c9965d754751.png)
